README.md

```diff
@@ -30,7 +30,7 @@ If you need help getting started with fogg, either open a github issue or join o
 You can use homebrew to install fogg –
 
 ```
-brew tap chanzuckerberg/homebrew-fogg
+brew tap chanzuckerberg/tap
```

Comment:
The repo is `chanzuckerberg/homebrew`. Does homebrew special-case this situation?

Reply:
yeah, moved it to `chanzuckerberg/hombrew-tap` because of some naming conventions they have (repo names have to be hombrew-* to be able to specify a short path)

Reply:
https://github.com/chanzuckerberg/homebrew-tap